contracts/tokens/BaseToken.sol

```diff
@@ -1,18 +1,16 @@
 // SPDX-License-Identifier: Mozilla Public License 2.0
 pragma solidity ^0.8.17;
 
+import "../CommunityOwnable.sol";
```

Comment:
Same here:

```suggestion
import { CommunityOwnable } from "../CommunityOwnable.sol";
```

To be fair, this file isn't using named imports at all yet, which I've fixed in #17 
I could merge it and then you can rebase on top of it.